### DISCUSSION TITLE
Refresh book count

### DISCUSSION THREAD

**Discussion by Hirkos97** [gotson/komga#1606] [Ideas]
Sometimes when you move things around in a library, you end up with “No Book” for number of books. Scans and reanalyzations do not always fix this requiring the addition of a temporary *cbz file to reset the count. 

If you click any of the series, you will see all the issues. It is strictly the count is off.

It be nice if if you could kick off a process that scanned for the count in these cases.

**Comment by gotson** [comment]
You can do a deep scan to fix this.

**Reply by Hirkos97** [reply]
Deep scan has not always fixed this issue. I often end up having to delete and re-add the library. That aside, It was a “nice to have” if it was an easy enhancement.

It usually happens if I move a folder that was previously scanned. Metadata and such will scan, the books are all accessible, but the count shows as “no books”.

I don’t have a current example as I readded my library but if I can get it to happen and also get a log, I will attach.

I very much appreciate this app and the work that is put into it. thank you for all you do!
